docs/sphinx/dev_guide/ci_tasks.rst

```diff
@@ -157,6 +157,128 @@ annotate the job for this. For example:
    describe the change in the ``RAJA/.gitlab/jobs/<MACHINE>.yml`` file where
    the job is overridden.
 
+
+Building the Intel clang + SYCL HIP compiler for use in CI
+------------------------------------------------------
+
+The SYCL CI tests on corona rely on a custom Intel Clang SYCL compiler that we 
+build ourselves. This compiler lives in the ``/usr/workspace/raja-dev/`` folder so 
+that it can be accessed by the gitlab CI system. Since the intel compiler does
+not do releases in the typical sense (they simply update their repo *every night*), 
+it may become necessary to periodically build a new version of the compiler to 
+ensure that we are using the most up-to-date version available. The steps for 
+building, installing, and running are shown here.
+
+Building the Compiler
+^^^^^^^^^^^^^^^^^^^^^
+
+.. important:: Because intel updates their compiler repo daily, there is a nonzero possibility that the head of the sycl branch will fail to build. 
+  In the event that it does not build, try checking out a different commit. On the intel/llvm GitHub page, one can see which of their 
+  commits builds by checking the status badge next to each commit. Look for a commit that passes. 
+
+#. Load the version of GCC that you want to use. in this case, we are using LC's gcc/10.3.1-magic installation. 
```

Comment:
I think all list items need to be indented from the main text so that the numbers generate properly. When they are at the same level as the main text, each item has number '1'. Please see https://raja.readthedocs.io/en/long58-ci_expansion/sphinx/dev_guide/ci_tasks.html

Reply:
Yeah, I saw that, I was in the process of changing it. It took a little bit for it to update after I pushed my changes.

Reply:
I don't recall what the exact rules are. If you run into something, there is likely an example elsewhere in the docs where it is done correctly. Not the most helpful comment, but that's often how I fix problems in the docs.